src/server/service/achievements.ts

```diff
@@ -80,11 +76,51 @@ const beginnersLuckAchievement: Achievement = {
   },
 };
 
+const realHumanAchievement: Achievement = {
+  name: "Real Human",
+  description: "First time played as a verified human",
+  calculate: ({ playerHistory, player }) => {
+    if (!playerHistory || !player.isVerified) return false;
+    return !alreadyReceivedAchievement(
+      player.userId,
+      playerHistory,
+      "realHuman",
+    );
+  },
+};
+
 export const MATCH_ACHIEVEMENTS: Record<AchievementId, Achievement> = {
-  "11": lastOneAchievement,
-  "12": goodBustAchievement,
-  "13": doubleAgentAchievement,
-  "101": busterStreakAchievement,
-  "201": firstTimerAchievement,
-  "202": beginnersLuckAchievement,
+  goodBust: goodBustAchievement,
+  doubleAgent: doubleAgentAchievement,
+  busterStreak: busterStreakAchievement,
+  firstTimer: firstTimerAchievement,
+  beginnersLuck: beginnersLuckAchievement,
+  realHuman: realHumanAchievement,
+};
```

Comment:
Move this at the top of the file

Reply:
This can't be done since the function are being called before they are defined.